### PR TITLE
Add llama3.3-70b-instruct model configs, tests and HF golden logits

### PR DESCRIPTION
**Pull request by hengtaoguo**
# Description

Add support for [llama3.3-70b-instruct](https://huggingface.co/meta-llama/Llama-3.3-70B-Instruct) for maxtext. This model is of the same architecture of llama3.1-70b. 
- Upload golden logits generated from HuggingFace. 
- 1_test_llama3.3_70b.sh for downloading HuggingFace checkpoings from GCS and convert to Maxtext format.
- 2_test_llama3.3_70b.sh for decoding, training and golden logits comparison. Including pre-converted checkpoints.
- Add llama3.3_70b to llama checkpoint conversion.

FIXES: [b/395908473](https://b.corp.google.com/issues/395908473)

# Tests
2_test: KL divergence below threshold ([screenshot](https://screenshot.googleplex.com/3d2FnRZFLLf3WXc)).

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [X] I have performed a self-review of my code.
- [X] I have necessary comments in my code, particularly in hard-to-understand areas.
- [X] I have run end-to-end tests tests and provided workload links above if applicable.
- [X] I have made or will make corresponding changes to the doc if needed.

### CODE REVIEW

File: end_to_end/tpu/llama3.3/70b/2_test_llama3.3_70b.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+# We defined path to unscanned checkpoint created in 1_test_llama3.3_70b.sh
+export UNSCANNED_CKPT_PATH=${BASE_OUTPUT_PATH}/${RUN_NAME}/checkpoints/0/items
+
+# To use our pre-converted checkpoint from huggingface Llama3.3-70B-Instruct, uncomment below two lines to override
+# export CONVERTED_CHECKPOINT=gs://maxtext-model-checkpoints/llama3.3-70b-instruct/2025-02-15-07-58/scanned/0/items
+# export UNSCANNED_CKPT_PATH=gs://maxtext-model-checkpoints/llama3.3-70b-instruct/2025-02-15-07-58/unscanned/checkpoints/0/items
+
+# TODO(mohitkhatwani): Fix XLAResourceExhaustion when loading unscanned model
+# # We run decoding on the `UNSCANNED_CKPT_PATH` for efficient decoding on the unscanned version of the checkpoint. Note that this checkpoint only has parameters and no optimizer state. 
+# # So, we use it by specifying`load_parameters_path=${CONVERTED_CHECKPOINT}`
```
Comment (khatwanimohit) [inline end_to_end/tpu/llama3.3/70b/2_test_llama3.3_70b.sh:49]: Can you uncomment this and see if decode works with unscanned ckpt ?

Comment (hengtaoguo) [reply end_to_end/tpu/llama3.3/70b/2_test_llama3.3_70b.sh:49]: This is still giving a "RESOURCE_EXHAUSTED" runtime error: [link](https://pantheon.corp.google.com/logs/query;cursorTimestamp=2025-02-26T22:11:46.129615567Z;duration=P1D;query=resource.type%3D%22k8s_container%22%0Aresource.labels.project_id%3D%22tpu-prod-env-one-vm%22%0Aresource.labels.location%3D%22us-east5%22%0Aresource.labels.cluster_name%3D%22bodaborg-v6e-256-dnd-yucmhab%22%0Aresource.labels.namespace_name%3D%22default%22%0Aresource.labels.pod_name:%22htg-llama33-decode-unscanned-slice-job-0-0-%22%20severity%3E%3DDEFAULT%0Atimestamp%3D%222025-02-26T22:11:46.129615567Z%22%0AinsertId%3D%22pelmymjqu0fbdxyz%22;storageScope=project?project=tpu-prod-env-one-vm). So should we keep those lines commented for now?

**Review by khatwanimohit — APPROVED** [review]
